src/runtime/GlobalObjectBuiltinError.cpp

```diff
@@ -134,7 +138,15 @@ void GlobalObject::installError(ExecutionState& state)
 
     // http://www.ecma-international.org/ecma-262/5.1/#sec-13.2.3
     // 13.2.3 The [[ThrowTypeError]] Function Object
-    m_throwTypeError = new NativeFunctionObject(state, NativeFunctionInfo(state.context()->staticStrings().ThrowTypeError, builtinErrorThrowTypeError, 0, NativeFunctionInfo::Strict));
+    m_throwTypeError = new NativeFunctionObject(state, NativeFunctionInfo(AtomicString(), builtinErrorThrowTypeError, 0, NativeFunctionInfo::Strict));
+    m_throwTypeError->markThisObjectDontNeedStructureTransitionTable();
```

Comment:
`setGlobalIntrinsicObject` could replace `markThisObjectDontNeedStructureTransitionTable` at here.
`setGlobalIntrinsicObject` is used for initialization of each intrinsic object.

Reply:
I changed it! thanks